framework/src/components/cache/defaults/config.js

```diff
@@ -3,37 +3,39 @@ const defaultConfig = {
 	properties: {
 		enabled: {
 			type: 'boolean',
-			default: false,
 			env: 'LISK_CACHE_ENABLED',
 		},
 		host: {
 			type: 'string',
 			format: 'ipOrFQDN',
-			default: '127.0.0.1',
 			env: 'LISK_REDIS_HOST',
 			arg: '-r,--redis',
 		},
 		port: {
 			type: 'integer',
 			minimum: 1,
 			maximum: 65535,
-			default: 6380,
 			env: 'LISK_REDIS_PORT',
 		},
 		db: {
 			type: 'integer',
 			minimum: 0,
 			maximum: 15,
-			default: 0,
 			env: 'LISK_REDIS_DB_NAME',
 		},
 		password: {
 			type: ['string', 'null'],
-			default: null,
 			env: 'LISK_REDIS_DB_PASSWORD',
 		},
 	},
 	required: ['enabled', 'host', 'port', 'db', 'password'],
+	default: {
+		enabled: false,
+		host: '127.0.0.1',
+		port: 6380,
+		db: 0,
+		password: null,
```

Comment:
does this have to be defined? or just backward compatibility?
Ideally it can be optional right?

Reply:
I just kept it exactly the same as before to avoid any unseen issue. 

https://github.com/LiskHQ/lisk/blob/0d2df7d376030122faae763a936e5a1a76f066bf/config/default/config.json#L27

It depends on redis library implementation how it treat this value. I believe it's just omitted if null.